packages/x-internals/src/css/css.ts

```diff
@@ -0,0 +1,10 @@
+import type { CSSObject } from '@mui/system';
+
+export function css<T extends Record<string, CSSObject>>(
+  _prefix: string,
+  _styles: T,
+): {
+  [K in keyof T]: string;
+} {
+  throw new Error(`The \`css()\` utility should never be called.`);
```

Comment:
This will be called when the Babel plugin is missing / malfunctioning?
Could you mention this in the error to avoid confusion? Because we do call `css()` in our codebase 😅

```suggestion
  throw new Error(`This function body should never be called. \`css()\` calls are transpiled by \`babel-plugin-mui-css\` both in runtime and build time.`);
```